packages/react-devtools-inline/webpack.config.js

```diff
@@ -73,6 +73,10 @@ module.exports = {
       __EXTENSION__: false,
       __PROFILE__: false,
       __TEST__: NODE_ENV === 'test',
+      // TODO: Should this be feature tested somehow?
+      __IS_CHROME__: false,
+      __IS_FIREFOX__: false,
+      __IS_EDGE__: false,
```

Comment:
Ugh, `react-devtools-inline` is shipped as an npm package, so these variables don't make much sense for it. Instead, it should be checked at runtime, not build time.

Will update this later

Reply:
I'm not quite sure why more of these weren't reached and subsequently errored before. Not sure why just the ones I added mattered.